source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,330 @@
+`set-pipeline` command.  To apply changes, run the following command
+from the project root:
+
+```
+fly set-pipeline --pipeline <pipeline-name> --config concourse/pipeline.yml
+```
+
+Committing the changes to master will not update the pipeline
+automatically.
+
+GDS Reliability Engineering provide [further
+documentation][big-concourse-docs].  More information on working with
+Concourse and the Fly CLI can be found in [the official
+documentation][concourse-docs].
+
+### Zero-Downtime Deployment
+
+The deployment tasks use the CloudFoundry v3 API commands in order to
+support zero-downtime deployment and various new features.
+
+When working with the cf cli tool, use the `v3` prefixed commands.
+For example: `cf v3-apps`.
+
+### Cancelling a failed deployment
+
+Occasionally a zero-downtime deployment may fail.  This can occur if
+the application fails to start, or the required number of app
+instances cannot be created in time.  To cancel and roll back to the
+previous deployment, run: `cf v3-cancel-zdt-push <app-name>`.
+
+### Links to Staging
+
+- [`govuk-coronavirus-business-volunteer-form`](https://govuk-coronavirus-business-volunteer-form-stg.cloudapps.digital/medical-equipment)
+- [`govuk-coronavirus-vulnerable-people-form`](https://govuk-coronavirus-vulnerable-people-form-stg.cloudapps.digital/live-in-england)
+
+These are protected by HTTP basic auth.  You can find the credentials
+in govuk-secrets and get them by running:
+
+```
+cd ~/govuk/govuk-secrets/pass
+./edit.sh 2ndline coronavirus-forms/creds
+```
+
+## Monitoring
+
+Grafana (there's a separate Grafana to look at) *TODO: where?*
+
+There's a [Splunk dashboard][splunk] for both of these services.  To
+access Splunk, you need to have the `GDS-006-GOVUK` permission on your
+Google account.  To get this permission, raise an IT Helpdesk ticket
+and post in `#cyber-security-help` to get them to confirm the request
+is legitimate.
+
+Application errors are sent to the GOV.UK hosted Sentry:
+
+- [`govuk-coronavirus-business-volunteer-form`](https://sentry.io/organizations/govuk/issues/?project=5172100)
+- [`govuk-coronavirus-vulnerable-people-form`](https://sentry.io/organizations/govuk/issues/?project=5170680)
+
+These are under the GOV.UK Sentry account and you can sign in with
+your Google Account.
+
+Logs for both applications are streamed to Logit and can be found in
+following dashboards:
+
+GOV.UK Production Corona Forms *TODO: where?*
+GOV.UK Staging Corona Forms *TODO: where?*
+
+## Extracting form responses (business only)
+
+Cabinet Office will require data exports at regular intervals.
+
+There is a rake task to export the data for a single day in JSON
+format.  This can then be sent to the relevant team (TODO: who?).
+
+```
+cf v3-ssh govuk-coronavirus-business-volunteer-form
+$ /tmp/lifecycle/shell
+$ rake export:form_responses["<date>"]
+```
+
+Date to be included in the format 2020-03-26.
+
+
+## Troubleshooting
+
+### What things will call you
+
+*TODO: what?*
```

Comment:
The GOV.UK PagerDuty on-call/2nd line pager will page you should these applications have issues.

Reply:
I assumed this section would be more "what sort of things will call you"

Reply:
```suggestion
The GOV.UK PagerDuty will page on-call/2ndline if these applications go down. It's connected up to a Pingdom check in the GOV.UK account that checks if the first form page is up.
```